continuous_integration/recipe/meta.yaml

```diff
@@ -48,6 +48,9 @@ requirements:
     - pygments >=2.7.1
     - tabulate
 
+  ignore_run_exports:
```

Comment:
Does this have to go in the top level `build` key? https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#export-runtime-requirements